### ISSUE TITLE
Body text color is too low-contrast

### ISSUE THREAD

**Issue by cfarm** [girldevelopit/gdi-new-site#98]
The font color for the paragraph text is too light and therefore inaccessible for visually-impaired users. See the WebAIM Color contrast tool here: http://webaim.org/resources/contrastchecker/

Font color is #7a7c7c, on background #ffffff which fails the checker.

Solution would be to choose a darker grey that passes the tests on the color checker tool.

**Comment by snkahn** [comment]
I don't have a copy of the build running, but I suggest #2d2e2e

**Comment by kstack7** [comment]
want me to apply it to my build? In a new branch of course :)

**Comment by juliaelman** [comment]
Fixed with PR #114 - thanks @minamarkham!

**Comment by cfarm** [comment]
Thanks @minamarkham!
